### PR TITLE
Room users trade user context

### PR DESCRIPTION
**Pull request by shotgunbooter**

### CODE REVIEW

**Comment by shotgunbooter** [comment]
I present to you

My pointless and probably not useful, feature :)

**Comment by b7c** [comment]
I think you forgot to push some local changes, I don't see what you posted here

**Comment by shotgunbooter** [comment]
> I think you forgot to push some local changes, I don't see what you posted here

COORRECT, i will push it

File: src/Xabbo/ViewModels/Room/Avatars/RoomAvatarsViewModel.cs [hunk 1]
```diff
@@ -184,6 +185,7 @@ avatar.Type is AvatarType.User &&
 
         FindAvatarCmd = ReactiveCommand.Create(FindAvatar, hasSingleContextAvatar);
         CopyAvatarToWardrobeCmd = ReactiveCommand.Create(CopyAvatarsToWardrobe, hasAnyContextUser);
+        TradeUserCmd = ReactiveCommand.Create(TradeUser, hasAnyContextUser);
```
Comment (b7c) [inline src/Xabbo/ViewModels/Room/Avatars/RoomAvatarsViewModel.cs:188]: just change this to `hasSingleContextUser` to make sure only a single user is selected to trade

File: src/Xabbo.Avalonia/Views/Room/RoomAvatarsView.axaml [hunk 2]
```diff
@@ -77,6 +77,11 @@
               IconSource="{icm:SymbolIconSource Symbol=BackpackAdd}"
               Text="Add to wardrobe"
               Command="{Binding CopyAvatarToWardrobeCmd}"
+            />
+              <ui:MenuFlyoutItem
```
Comment (b7c) [inline src/Xabbo.Avalonia/Views/Room/RoomAvatarsView.axaml:81]: you can also fix indentation here

**Review by b7c — CHANGES_REQUESTED** [review]

**Review by b7c — APPROVED** [review]

**Comment by b7c** [comment]
good job

**Pull request merged**